webtool/test_fetch.py

```
@@ -807,3 +810,61 @@ def test_kaputter_netloc_ist_eine_vorbedingung_kein_pip(projekt, monkeypatch):
     assert versuche == []
     with pytest.raises(fetch.Vorbedingung, match="URL"):
         fetch.check_url("https://[::1")
+
+
+# --- Sprecherzahl je URL (#Vorschau beim Hinzufuegen) ------------------------
+
+def test_download_one_traegt_die_sprecherzahl_der_position_ein(projekt):
```

Ruff (0.16.1)

[warning] Unused function argument: `projekt`

(ARG001)